bin/build-dist

```diff
@@ -15,6 +15,17 @@ elif [[ -e minil.toml ]]; then
     # shellcheck disable=SC2012
     DIR=$(ls -td -- */ | head -n 1 | cut -d'/' -f1)
     mv "$DIR" build_dir
+elif [[ -e Build.PL ]]; then
+    rm -rf "$BUILD_DIR"
+    perl Build.PL
+    ./Build clean
+    ./Build manifest
+    ./Build distdir
+
+    # Module::Build does not support a build directory, so we have to find
+    # the last created dir and use that.
+    DIR=$(ls -td -- */ | head -n 1 | cut -d'/' -f1)
```

Comment:
A possible alternative here is `"$(perl -MModule::Build -e'print Module::Build->resume->dist_dir;')"`. I'm not sure if I actually prefer it though.

Reply:
Thanks for the suggestion. I took this from the existing code for minila. It seemed sensible, but I'm not very familiar with Module::Build, just shaving a yak here really. I'm happy to defer to whatever you think works best.

Reply:
I like both, but I think the solution @haarg suggested makes it more immediately obvious to me what's happening.

Reply:
I've gone with the suggestion. I agree with @oalders that using MB to get the dir out rather than fiddling with the file system is much cleaner, even if it's technically slower.